alu/mul/columns.lisp

```diff
@@ -37,7 +37,7 @@
     (BYTE_H_0 :byte@prove)    (ACC_H_0 :i64)
     ;==========================
     (EXPONENT_BIT               :binary@prove)
-    (EXPONENT_BIT_ACCUMULATOR   :i8)
+    (EXPONENT_BIT_ACCUMULATOR   :i128)
```

Comment:
This columns shouldn't be more than 1 byte wide ... Which ref tests it was breaking ?